server.js

```diff
@@ -8,31 +8,34 @@ const express = require('express');
 // origin - the beginning of your url
 const cors = require('cors');
 const PORT = process.env.PORT || 3001;
-const key = process.env.WEATHER_API_KEY
+const key = process.env.WEATHER_API_KEY;
 // const weatherData = require('./data/weather.json');
 
 // singleton ( there can only be one!! )
 const app = express(); // returns an object, with methods designed to handle Requests.
+app.use(cors()); // set up cross origin resource sharing
 // enable cross origin resource sharing between localhost:3001 and any other url that may make a request.
 // app.use(cors());
 
-// class Forecast {
-//   constructor(obj) {
-//     this.date = obj.datetime;
-//     this.description = 'low of ' + obj.low_temp + ', high of ' + obj.high_temp + ' with ' + obj.weather.description.toLowerCase();
-//     this.searchQuery = '';
-//   }
-// }
+class Forecast {
+  constructor(obj) {
+    this.date = obj.datetime;
+    this.description = obj.weather.description;
+  }
+}
 
-app.use(cors()); // set up cross origin resource sharing
 app.get('/weather', (request, response) => {
   let searchQuery = request.query.searchQuery;
   console.log(searchQuery);
-  let weatherKey = `https://api.weatherbit.io/v2.0/current/search?key=${key}&query=${this.searchQuery}&format=json`;
+  let weatherKey = `https://api.weatherbit.io/v2.0/forecast/daily?key=${key}&city=${searchQuery}&days=5`;
   axios.get(weatherKey)
     .then(res => {
-      console.log(res.data.results);
-      response.send(res.data.results);
+      console.log(res);
+      let weatherArray = res.data.data.map(day => {
```

Comment:
This looks good, a good use of map. I think the next step is move the weather logic into into its own function, and then move Forecast class into its own file. In the forecasts file, you'll end it with "module.exports = Forecast" so that it will export.  Then after that, in this server file you can use "const Forecast= require('./Forecast.js')" to import it. With that, you can use it just like before, such as "return new Forecast(day)".